cms/djangoapps/contentstore/views/tests/utils.py

```diff
@@ -66,7 +66,7 @@ def validate_preview_html(self, xblock, view_name, can_add=True, can_reorder=Tru
         )
         self.validate_html_for_action_button(
             html,
-            'button class="btn-default edit-button action-button">',
+            'button class="btn-default edit-button action-button"',
```

Comment:
The removal of ">" here allows there to be other attributes on the edit button, which is where the flags appear

Reply:
Thanks for the explanation!